packages/yarnpkg-pnp/sources/loader/internalTools.ts

```diff
@@ -2,11 +2,11 @@ import {Path, npath} from '@yarnpkg/fslib';
 
 export enum ErrorCode {
   API_ERROR = `API_ERROR`,
-  BLACKLISTED = `BLACKLISTED`,
```

Comment:
This is breaking for anyone using the API directly and also changes the errors thrown at runtime